sdk/python/kfp/dsl/_container_op.py

```diff
@@ -24,7 +24,8 @@ class ContainerOp(object):
   """Represents an op implemented by a docker container image."""
 
   def __init__(self, name: str, image: str, command: str=None, arguments: str=None,
-               file_outputs : Dict[str, str]=None, is_exit_handler=False):
+               file_outputs : Dict[str, str]=None, output_artifact_paths : Dict[str, str]={},
```

Comment:
`={}` is dangerous. This dictionary is only created once and then shared between all instances. It's better to have `None` default and then check for it and assign a new empty dict.